### ISSUE TITLE
bazel test command broken because of Missing ROCM input files.

### ISSUE THREAD

**Issue by claynerobison** [tensorflow/tensorflow#36343]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04 tensorflow/tensorflow:devel-py3 container
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): source
- TensorFlow version: commit ID 720b16121ea60d914f2d3df47e580e7e1e5542ae or latest master
- Python version: Python 3.6.9
- Installed using virtualenv? pip? conda?: Not installsed
- Bazel version (if compiling from source): 1.2.1
- GCC/Compiler version (if compiling from source): gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the problem** 
Unable to complete bazel test command because of missing rocm input files. 

```
[eigen.log.zip](https://github.com/tensorflow/tensorflow/files/4135491/eigen.log.zip)

ERROR: missing input file '@local_config_rocm//rocm:rocm/include/hipcub/hipcub_version.hpp'
ERROR: missing input file '@local_config_rocm//rocm:rocm/include/rocprim/rocprim_version.hpp'
ERROR: /root/.cache/bazel/_bazel_root/104a68818e52204e40c368960e574528/external/local_config_rocm/rocm/BUILD:119:1: @local_config_rocm//rocm:rocprim: missing input file '@local_config_rocm//rocm:rocm/include/hipcub/hipcub_version.hpp'
ERROR: /root/.cache/bazel/_bazel_root/104a68818e52204e40c368960e574528/external/local_config_rocm/rocm/BUILD:119:1: @local_config_rocm//rocm:rocprim: missing input file '@local_config_rocm//rocm:rocm/include/rocprim/rocprim_version.hpp'
ERROR: /host/tensorflow/tensorflow/python/kernel_tests/BUILD:1222:1: Creating runfiles tree bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/summary_v1_ops_test.runfiles failed: build-runfiles failed: error executing command 
  (cd /root/.cache/bazel/_bazel_root/104a68818e52204e40c368960e574528/execroot/org_tensorflow && \
  exec env - \
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
    PYTHON_BIN_PATH=/usr/local/bin/python \
    PYTHON_LIB_PATH=/usr/local/lib/python3.6/dist-packages \
    TF2_BEHAVIOR=1 \
    TF_CONFIGURE_IOS=0 \
    TF_ENABLE_XLA=1 \
  /root/.cache/bazel/_bazel_root/install/84defa6eb1e9416bf92d6f89ab2d4f31/build-runfiles bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/summary_v1_ops_test.runfiles_manifest bazel-out/k8-opt/bin/tensorflow/python/kernel_tests/summary_v1_ops_test.runfiles): Process terminated by signal 15: Process terminated by signal 15
ERROR: /root/.cache/bazel/_bazel_root/104a68818e52204e40c368960e574528/external/local_config_rocm/rocm/BUILD:119:1 2 input file(s) do not exist```

These files do not exist in the bazel cache.


**Provide the exact sequence of commands / steps that you executed before running into the problem**

```$ docker pull tensorflow/tensorflow:devel-py3
$ git clone https://github.com/tensorflow/tensorflow
$ docker run -ti --rm -v $(pwd):/host tensorflow/tensorflow:devel-py3
# cd /host/tensorflow
# yes "" | python configure.py
# bazel --nosystem_rc --nohome_rc test --cxxopt=-D_GLIBCXX_USE_CXX11_ABI=0 --copt=-march=haswell --copt=-mtune=broadwell --copt=-O3 --copt=-Wformat --copt=-Wformat-security --copt=-fstack-protector --copt=-fPIC --copt=-fpic --linkopt=-znoexecstack --linkopt=-zrelro --linkopt=-znow --linkopt=-fstack-protector --config=v2 --test_timeout 300,450,1200,3600 --test_env=KMP_BLOCKTIME=0 -s --cache_test_results=no --test_size_filters=small,medium,large,enormous -c opt -- //tensorflow/... -//tensorflow/compiler/... -//tensorflow/lite/... -//tensorflow/stream_executor/cuda/... -//tensorflow/python/autograph/pyct/... -//tensorflow/core/kernels:eigen_mkldnn_contraction_kernel_test```

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

log file attached

**Comment by claynerobison** [comment]
This problem started occuring sometime after the last commit on Jan 26: f88440725cdf3808a20238864cfe75a5cd9f2f22 because I'm able to run the bazel test command above just fine against that commit.

**Comment by claynerobison** [comment]
I can't figure out why this command is trying to build rocm components...

**Comment by claynerobison** [comment]
@angerson This is the culprit: b105944eb6c563849a085a1765d6700ee2c0f35c in the tensorflow/core/kernels/BUILD file. The test command succeeds on the commit before this: 0a3c2988804a71371f51e182bd70b81a0d88be6d.

**Comment by Bengt** [comment]
I can confirm this. I get this issue with with the latest tensorflow-rocm-upstream, into which tensorflow was last merged on Friday. After checking out the earlier commit @claynerobison pointed at, I do no longer see this error. I am not sure if this is the correct issue tracker, though. Pinging @whchung and @sunway513 for awareness.

**Comment by ekuznetsov139** [comment]
This appears to be fixed by the commit 930c651.
